cloud/azure/bin/deploy

```diff
@@ -1,7 +1,27 @@
 #! /usr/bin/env bash
 
-KEEP_ORIGINAL_PWD=true source "${PROJECT_BASE_DIR}/bin/lib.sh"
-source "${PROJECT_BASE_DIR}/cloud/shared/bin/lib.sh"
-source "${PROJECT_BASE_DIR}/cloud/azure/bin/lib.sh"
+source "cloud/shared/bin/lib/health.sh"
```

Comment:
i think this lives in shared with the set-up script instead of just bin/

Reply:
this also implements just the application update vs the terraform changes, is the plan for it to include terraform changes?

Reply:
so I have this script being called by the deploy scripts in dev_azure and staging_azure, since calling terraform will look different in dev and staging. tbh I'm pretty lost on how to structure this code so we should probably meet and work on it together at some point